Software/Development/SemesterProject/App/dutyCycle.c

```diff
@@ -21,92 +21,92 @@
 //
 // Quasi-global variables definition
 //
-static struct dutyCycleTag
+/*static struct dutyCycleListTag
```

Comment:
why having commented parts?

Reply:
True, i forgot to take it out, corrected.